tests/tests/features-kubernetes.yaml

```
@@ -26,6 +26,17 @@
     project: ci-features-{{ lookup('env','CLUSTER_TYPE') }}
     branch: ingress-annotations
 
+- include: features/short-router-url.yaml
+  vars:
+    testname: "SHORT ROUTER URL {{ cluster_type|upper }}"
+    git_repo_name: features.git
+    project: ci-features-{{ cluster_type }}
+    branch: short-router-url-from-a-very-long-environment-name-like-this
+    # lagoon_environment is truncated for very long branches.
+    # See commons/src/tasks.ts
+    lagoon_environment: short-router-url-from-a-very-l-ebe8
+    check_url: "http://node.{{ project | regex_replace('_', '-') }}.{{ branch | regex_replace('/', '-') }}.{{ route_suffix }}"
```

```suggestion
    check_url: "http://node.{{ project | regex_replace('_', '-') }}.{{ branch | regex_replace('/', '-') }}.{{ lookup('env','ROUTE_SUFFIX_HTTP') }}:{{ lookup('env','ROUTE_SUFFIX_HTTP_PORT') }}"
```
Variable differences between 1.x and 2.x